dac_calibration_server.py

```diff
@@ -349,7 +349,7 @@ def correct_analog(self, c, data, average_ends=False, dither=False):
 
     @setting(33,
         'Correct Analog FT',
-        data=['*v: Single channel data'],
+        data=['*c: Single channel data'],
```

Comment:
I definitely wouldn't put units on this. The time step should be specified separately and all sequence data should be dimensionless. Anyway, I'm fine to not change the type tag but I can't imagine how it would cause problems to change it to explicitly disallow units.